genrl/deep/agents/base.py

```diff
@@ -4,19 +4,26 @@
 import numpy as np
 import torch
 
-from genrl.deep.common.utils import load_params, set_seeds
+from genrl.deep.common.utils import set_seeds
 
 
 class BaseAgent(ABC):
-    def __init__(self, network_type: str, env: Any, epochs: int = 100, **kwargs):
+    def __init__(
+        self,
+        network_type: str,
+        env: Any,
+        batch_size: int = 64,
+        gamma: float = 0.99,
+        layers: Tuple = (64, 64),
+        **kwargs
+    ):
         self.network_type = network_type
         self.env = env
-        self.epochs = epochs
+        self.batch_size = batch_size
+        self.layers = layers
+        self.gamma = gamma
         self.seed = kwargs.get("seed", None)
         self.render = kwargs.get("render", False)
```

Comment:
No need to do a "get" operation, defeats the purpose of having ** in `__init__`

Reply:
Ok, it was already there. I'll change it.